src/plugin/log/src/Logger.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "spdlog/spdlog.h"
+
+namespace ES::Plugin::Log {

```

Comment:
logger is a plugin??

Reply:
It's more like utils function you're right

Reply:
Done